documentation/docs/02_account_manager.md

```diff
@@ -76,6 +77,11 @@ Initialise the wallet with a randomly generated mnemonic and a given node.
 $ ./wallet init --node "http://localhost:14265/"
 ```
 
+Initialise the wallet with a given coin type.
+```sh
+$ ./wallet init --coin-type "iota"
```

Comment:
I think shimmer is better here as default example, because that's what most people should probably use as long it's not compatible with the mainnet

Reply:
Yeah I just thought I'd put something different if it's explicit as shimmer is the default, but I changed it anyway